tests/lib/rules/post-message-no-wildcard-targets.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @fileoverview Disallow using the wildcard `*` in postMessage
+ * @author Automattic
+ * @copyright 2017 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require( '../../../lib/rules/post-message-no-wildcard-targets' ),
+	config = { env: { es6: true } },
+	RuleTester = require( 'eslint' ).RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+( new RuleTester( config ) ).run( 'post-message-no-wildcard-targets', rule, {
+	valid: [
+        { code: 'foo()' },
+        { code: 'foo( 1, \'*\' )' },
+        { code: 'postMessage( 1, \'test\' )' },
+        { code: 'postMessage( null, test )' },
+        { code: 'postMessage( true, test() )' }
+	],
+
+	invalid: [
```

Comment:
Seems fine after testing it myself, but might be worth adding a test for real-world example where postMessage is called on a window object, e.g. `frame.postMessage( 'bob', '*' )`